met-web/src/components/common/Typography/Headers.tsx

```diff
@@ -35,7 +36,6 @@ export const Header1 = ({
                 marginBottom: '2rem',
                 marginTop: '1.5rem',
                 fontWeight: fontWeight(weight),
-                color: '#292929',
```

Comment:
Are we using static color codes now or should these be assigned variables?

Reply:
I'm transitioning to using MUI theming for certain properties instead of hardcoded colors, allowing for dark theme implementations, high contrast modes, etc.